modules/Broadcast.js

```diff
@@ -40,6 +40,7 @@ const createBroadcast = (initialState) => {
 class Broadcast extends React.Component {
   static propTypes = {
     channel: PropTypes.string.isRequired,
+    compareFunction: PropTypes.func.isRequired,
```

Comment:
This shouldn’t be `.isRequired` — that means that everyone has to provide the prop.

Reply:
It's provided in `defaultProps` so that users need only include if they desire. It is most definitely required though as Broadcast will break without it.

Reply:
I made a `codesandbox` just to be sure. 😅 https://codesandbox.io/s/rj2v960jn -- Thanks for bringing that up.

Reply:
Cool, TIL!